wectl/tests/test_integration.py

```
@@ -0,0 +1,233 @@
+    """Test namespace functionality."""
+
+    def test_list_with_namespace(self, runner):
+        """Test listing resources with namespace flag."""
+        result = runner.invoke(cli, ["get", "ghosts", "-n", "default"])
+        # Accept success or auth error
+        if result.exit_code != 0:
+            assert "Error" in result.output
+
+    def test_list_with_nonexistent_namespace(self, runner):
+        """Test listing from non-existent namespace."""
+        result = runner.invoke(cli, ["get", "ghosts", "-n", "nonexistent-ns-12345"])
```

Ruff (0.14.6)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)